Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -28,6 +28,7 @@
 				2DA8152C28A4204400CF811C /* PBXTargetDependency */,
 				BC8283D516B4C01F00A278FE /* PBXTargetDependency */,
 				2D9FB22C2375245C0049F936 /* PBXTargetDependency */,
+				07275C6B2D00ED43002315A5 /* PBXTargetDependency */,
```

Comment:
This aggregate target is what gets built by Apple's internal build system. You can't list your target here without causing overlapping content. Instead, put it in the "All" aggregate target like we do for WebKitSwift.